### ISSUE TITLE
Read-only properties show in request body

### ISSUE THREAD

**Issue by adnweedon** [scalar/scalar#4040]
### What happens?

If I make a property in a schema read only, it stil shows when that schema is used as a request body, even though the property isn't relevant:
![image](https://github.com/user-attachments/assets/de122ea9-8e89-45ec-83ab-79c9516299ca)

I have made an example using the Pet Shop example spec, where I have made the Pet id field read only - note that it still shows for the Pet POST endpoint.

### What did you expect to happen?

I expected the read only properties not to be shown in the Body section, because they aren't relevant.

### OpenAPI Document

https://sandbox.scalar.com/#tag/pet/POST/pet

**Comment by hanspagel** [comment]
Good catch! I think that was the case until recently, definitely an issue.

**Comment by hanspagel** [comment]
Okay, I finally found time to test this, sorry for the super long wait. It turns out, we’re doing all we can, but those attributes are just not tagged as `read-only`/`write-only`.

I’m adding this to our example in #4477.
